assets/js/src/newsletter_editor/initializer.jsx

```diff
@@ -15,7 +16,7 @@ const renderBreadcrumb = (newsletterType) => {
     'editor'
   );
 
-  ReactDOM.render(breadcrumb, breadcrumbContainer);
+  ReactDOM.render(<MemoryRouter>{breadcrumb}</MemoryRouter>, breadcrumbContainer);
```

Comment:
I don't understand why are you using `MemoryRouter` here. Actually any `Link` or `Redirect` inside here will not work!

Reply:
Exactly, `Link` doesn't work without a `Router` it gives a warning in the console now.